site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,208 @@
+[bunnei](https://github.com/bunnei) rewrote the Virtual Memory Manager (known internally as `Project Epimetheus`), which drastically reduced RAM use by games, and helped Multicore support (also known as`Project Prometheus`) become a reality.
+
+One of the features in yuzu, and a common one in other emulators, is the ability to separate the rendering thread from the main emulation thread. This makes better use of CPU's resources, while also helping deliver better performance.
+That is exactly what Asynchronous GPU does in yuzu. Thanks to [Blinkhawk](https://github.com/FernandoS27), our Emulated GPU improved drastically, achieving better performance, stability, and accuracy.
+These changes introduce the option to adjust the GPU accuracy level while playing.
+
+{{< imgs
+    "./31.png| Performance may be reduced with more accurate levels"
+  >}}
+
+{{< single-title-imgs
+    "Normal accuracy (left) and High accuracy (right) (Super Mario Odyssey)"
+    "./v1.mp4"
+    "./v2.mp4"
+  >}}
+
+
+## One Line to Fix 'em All
+
+`Toki Tori` had sound problems that were related to a floating point instruction, and while investigating the issue, 
+[Merry](https://github.com/MerryMage) and [bunnei](https://github.com/bunnei) were worried that it may be a problem in [Dynarmic](https://github.com/MerryMage/dynarmic), our ARM to x64 recompiler.
+Turns out the problem was just a bad default value in yuzu’s Dynarmic configuration.
+Remember what we said about bunnei and serendipity? With a simple one line change, we fixed tons of bugs on a lot of games! Special mention goes to `Pokémon Sword/Shield`, where thanks to this single line change, the game no longer softlocked and became fully playable!
+
+Here's a *partial* list of fixes:
+
++ Pokémon Sword/Shield - All softlocks fixed, running in tall grass fixed, hair salons fixed
++ The Legend of Zelda: Link's Awakening - Miniature placing fixed
++ Toki Tori, Final Fantasy 7, Diablo 3, Project DIVA MegaMix - Distorted audio fixed
++ Team Sonic Racing - Physics fixed
++ Onimusha Warriors, Tales of Vesperia, The Messenger - Game logic fixed
++ Skyrim - Audio looping fixed, game progresses further
++ Oninaki - Elevator softlock fixed
++ All Unreal Engine 4 games - Motion jitter fixed
++ Super Smash Bros. Ultimate - Fighter select jitter / flickers fixed
++ Starlink - Softlock fixed, game progresses further
+
+&nbsp;
+
+{{< imgs
+    "./32.png| This sofa is no longer our enemy (Pokémon Sword/Shield)"
+  >}}
+
+## Bug fixes and improvements
+[Morph](https://github.com/Morph1984) did some Kirby magic and implemented the first steps to get World of Light 
+working in `Super Smash Bros. Ultimate`. 
+Even thou it doesn’t work yet, the first step is always the most important one!
+
+[Kewlan](https://github.com/Kewlan) once again strikes with more UI quality of life improvements. 
+This time, more UI fixes to the placement of buttons mappings in the input settings, specifically this time for single joycons.
```

Comment:
Forgot about replacing that section with:

`[Kewlan](https://github.com/Kewlan) strikes once again with more quality-of-life UI improvements. This time, they've fixed the placement of button mappings in the input settings for single joycons.`